src/react-components/input/ToolbarButton.js

```diff
@@ -50,6 +51,7 @@ export const ToolbarButton = forwardRef(
         )}
         disabled={disabled}
         title={title}
+        onClick={onClick}
```

Comment:
Not in this PR but as the FE matures it might be good to remove these {...rest} catch alls, side note I know they where there, I just think they can be cleaver yet misleading, you are establishing these nice prototypes soon to be TS type and I think it would be nice to have concrete props being sent to the component with no wild card catch all.